plugins/k8s/resoto_plugin_k8s/deferred_edges/__init__.py

```diff
@@ -15,3 +16,4 @@ def create_deferred_edges(graph: Graph) -> None:
             continue
         link_all_aws(graph, node)
         link_all_do(graph, node)
+        link_all_azure(graph, node)
```

Comment:
I think we can improve this code:  we should detect the cloud we are running in and then only call the relevant link method. Can you try to come  up with a proposal?